pkgs/applications/graphics/rapid-photo-downloader/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv, fetchurl, python3Packages
+, file, intltool, gobjectIntrospection, libgudev
+, udisks, glib, gnome3, gst_all_1, libnotify
+, exiv2, exiftool, qt5, gdk_pixbuf
+}:
+
+python3Packages.buildPythonApplication rec {
+  name    = "rapid-photo-downloader-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed. Instead, use
```
pname = "rapid-photo-downloader";
```

Reply:
Yes, forget to do it for the main package as well. This detail could use some documentation in the Nixpkgs manual.